### ISSUE TITLE
When i fine tune the BERT on my serve, it always says Segmentation fault? 

### ISSUE THREAD

**Issue by bianximo** [huggingface/pytorch-transformers#368]
I have done some modification on BertForSequenceClassification to apply a mutilabel prediction task, but
when i run my code on my serve, it always says that Segmentation fault when it runs to 
"loss = model(input_ids, segment_ids, input_mask,label_ids)", and even if i don't use GPU and fp16, it comes the same fault.
but if i run the code on my PC ,it works well,just very slow....
what 's wrong??

**Comment by Benhawkins18** [comment]
I had the same issue. I'm getting seg fault on an aws deep learning ami with a tesla v100 gpu instance. I get the error with or with out fp16

**Comment by datduong** [comment]
I also have the same problem. Did you guys figure out any solution?

**Comment by thomwolf** [comment]
Can you try running the model:
- with a very small batch size to check if it's an OOM error
- with `CUDA_LAUNCH_BLOCKING=1` to see the exact line causing the error
?

**Comment by Benhawkins18** [comment]
setting the env variable CUDA_LAUNCH_BLOCKING=1 doesn't give me any additional error messages. I still see the same error. 
 
Tried it with batch size 1 and I get the same issue

**Comment by datduong** [comment]
I got the code to run on my school server. I updated the gcc using this `conda install -c psi4 gcc-5`.

**Comment by nategoodman376** [comment]
I am also experiencing this issue. Anyone else figure it out?
